spec/test-data/translations/glotpress-download/expected/values-fr/strings.xml

```diff
@@ -30,4 +30,27 @@ Language: fr
         <item>En cours…</item>
         <item>Toutes les commandes…</item>
     </string-array>
+    <string name="threat_fix_description">Les prix vont de 70–120 $ par heure</string>
+    <string-array name="settings_jetpackdescription">
+        <item>Vous pouvez marquer une adresse IP (ou une série d’adresses IP) comme « Toujours autorisé »</item>
+        <item>pour empêcher leur blocage par Jetpack. IPv4 et IPv6 sont acceptables.</item>
+        <item>Pour spécifier une plage, entrez la valeur basse et la valeur haute en les séparant par un tiret. Exemple : 12.12.12.0–12.12.12.100</item>
+    </string-array>
+    <string name="threat_fix_description_large">Les prix vont de 70 – 120 $ par heure</string>
+    <string-array name="settings_jetpackdescription_large">
+        <item>Vous pouvez marquer une adresse IP (ou une série d’adresses IP) comme « Toujours autorisé »</item>
+        <item>pour empêcher leur blocage par Jetpack. IPv4 et IPv6 sont acceptables.</item>
+        <item>Pour spécifier une plage, entrez la valeur basse et la valeur haute en les séparant par un tiret. Exemple : 12.12.12.0 – 12.12.12.100</item>
```

Comment:
Yep, good point. Since the Android linter doesn't cover this case though, I think we can live with this version and iterate in the future if required. 
I mean, while your suggestion about the attribute is good and, on the linter side, we could add the strings to the baseline (though, IIRC, this should be done for every language, so it's not a great solution), it's pretty easy to get side tracked and end up doing something very complex :-) For example: then, how do we handle the case where there are two ranges in a string and only one should be substituted? 😅